src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/IBulkContentHandler.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.ContentManagement.Handlers;
+
+public interface IBulkContentHandler
```

Comment:
Mentioned this in the upcoming release notes: https://docs.orchardcore.net/en/latest/releases/2.0.0/. See https://docs.orchardcore.net/en/latest/guides/contributing/contributing-documentation/.

Reply:
Done